packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Authorization Servers
+meta:
+  - name: description
+    content: An overview of what an authorization server is and the types of authorization servers available at Okta.
+---
+## What is an authorization server
+
+At its core, an authorization server is simply an engine for minting OpenID Connect or OAuth 2.0 tokens. An authorization server is also used to apply access policies. Each authorization server has a unique issuer URI and its own signing key for tokens to keep a proper boundary between security domains.
```

Comment:
Should we assume that the customers/readers are familiar with OAuth? Is it worth having OAuth 2.0 as a link?